adafruit_ht16k33/ht16k33.py

```diff
@@ -24,6 +51,7 @@ def _write_cmd(self, byte):
             self.i2c_device.write(self._temp)
 
     def blink_rate(self, rate=None):
```

Comment:
This could be a get/set property.

Reply:
I know I filed a separate issue for it: #2